src/jumio-transactions/jumio-transaction.service.ts

```diff
@@ -77,6 +79,40 @@ export class JumioTransactionService {
     });
   }
 
+  scrubExtractionData(
+    lastWorkflowFetch: JumioTransactionRetrieveResponse | undefined,
+  ): JumioTransactionRetrieveResponse | undefined {
+    if (lastWorkflowFetch === undefined) {
+      return undefined;
+    }
+    const cleanExtractions = [];
+    for (const extraction of lastWorkflowFetch.capabilities.extraction) {
+      cleanExtractions.push({
+        ...extraction,
+        data: {
+          type: extraction.data.type,
+          subType: extraction.data.subType,
+          issuingCountry: extraction.data.issuingCountry,
+          firstName: extraction.data.firstName,
+          lastName: extraction.data.lastName,
+          expiryDate: extraction.data.expiryDate,
+          dateOfBirth: '1600-01-01',
```

Comment:
crossing our fingers that [Friedrich Spanheim](https://en.wikipedia.org/wiki/Friedrich_Spanheim) doesn't KYC

Reply:
Lol how did you know this man was born on that date? What a random fact